modules/Testing-Snapshot--Goal-446/README.md

```diff
@@ -38,7 +38,7 @@ You will start by cloning a starter app the has full CRUD functionality. From he
 - All "End to End" and Integrations tests are setup such that the database tables are truncated, and seeded before each test run.
 - End to End testing: Test the http routes exposed in the server
   - You have `chai-http` installed
-  - A test for the `/` GET route to make sure that the correct page is getting rendered and all the contacts are being received from the database
```

Comment:
its possible and totally reasonable to have a test that:
1. insert users into the database
2. renders '/'
3. asserts that the page that was rendered contains the users we put in the database

Reply:
Yeah definitely. You can slack me. @somayab

Reply:
I'd love for @punitrathore to make the call on this one since its phase 3 curriculum content.